config/mime-multipart.go

```diff
@@ -0,0 +1,160 @@
+package config
+
+import (
+	"io"
+	"io/ioutil"
+	"mime"
+	"mime/multipart"
+	"net/mail"
+	"os"
+	"strings"
+
+	"github.com/coreos/coreos-cloudinit/pkg"
```

Comment:
Same deal here. Config should not import any other local packages from coreos-cloudinit - it should be completely standalone.